### ISSUE TITLE
TEDIT:  add meta-d for man page, meta-o for ED open

### ISSUE THREAD

**Issue by rmkaplan** [Interlisp/medley#1739]
When seeing a Tedit document that has the names of Lisp items (function, variable, property...), typing meta-D should open up the corresponding DINFO man page, if any), typing meta-O should open up the current editor (SEDIT, or TTY editor).  This would provide an experience consistent with Sedit.

**Comment by masinter** [comment]
who do you imatine would use this? What other fcilities would someone used to editing emacs or vscode need?  

if you want SEdit-like behaior, how agout a (read0only) SEdit imageobject for history and SEE from filebrowser etc...

**Comment by rmkaplan** [comment]
I imagine using this a lot--in fact I already do.  I use the filebrowser SEE command (and also the exec command ts (for "tedit-see" in ilispusers/TEDIT-PF-SEE) for scrollable examination of Lisp source files.  And also the command tf instead of pf for scrollable examination of individual functions that I want to look at without actually corrupting my environment by loading their files just to view their definition in the editor.

When looking at a piece of source code--or even when looking at the Tedit file that documents some code--this extension would make it easy to drill down (either man or ED) to get more information, with the same user-interface conventions as Sedit.  The current alternative is to find an exec, partially type-in a command, and shift-select.  Not very smooth.

This is not aimed at somebody who wants to edit the source files as text in some other environment (emacs , bbedit...).  It's for someone working within the Medley environment who wants to easily get the additional information that our integrated environment can provide.

To put it another way, this is not about getting Sedit-like behavior, in terms of editing.  It's about bridging between the textual representation of Lisp files and definitions and other system facilities.

**Comment by rmkaplan** [comment]
Addressed in #1857
